src/components/TodoList/TodoItem.jsx

```diff
@@ -5,8 +5,30 @@ import axios from "axios";
 
 const TodoItem = (todoItem) => {
     const todoItemData = todoItem.todoItem;
-    const [gradientColor, setGradientColor] = useState("from-yellow-100");
-    const [borderColor, setBorderColor] = useState("border-black");
+    const priority = todoItemData.priority;
+
+    const todoContainerColor = {
```

Comment:
This object can be defined before the component so it is not declared on every render of the component